scripts/checkpatch.sh

```diff
@@ -24,7 +24,9 @@ hash $CHECKPATCH 2>/dev/null ||
 usage() {
   SCR=$(basename "$0")
   echo "Usage: $SCR [--working]                 Check working area"
-  echo "       $SCR <commit>...                 Check specific commit(s)"
+  echo "       $SCR <commit>...                 Check specific commits,
+                                                symbolic names, and/or revision
+                                                selections"
```

Comment:
Would few examples help user to understand this extended syntax, specifically regarding revision selections?

Reply:
I think that would be helpful. I was also thinking of reworking the entire help message into a more readable format. Maybe something that looks like the checkpatch.pl help message.